### PR TITLE
feat: updated Readme.md + fixing application node bug

### PR DESCRIPTION
**Pull request by SumantxD**
* demonstrated how to operate the simulator GUI 
* fixed the error with the subscribe node that occurred while creating flow elements from the yaml file

### CODE REVIEW

**Comment by NektariosFifes** [comment]
hey as you can see in the pull request
there are commits that have already been merged
so this commit overides them
after a new pull request is merged in the main asyncapi/simulator repo make sure to
1 update the master branch in your fork
2 rebase the branch in your fork with the master of your fork
3 or delete the branch as it has been merged and create a new one
I think it is important for our repo to have a clean commit history so as to not get things confused later on. It is a good habit cause when things get more complicated there might be issues where we cant revert , etc...

so your latest pull request must have only the 2 latest commits you made
you can do that by updating your fork , fetching your fork locally with git fetch (--all)
then from your local master branch you need to create a new branch then copy the changes from your current branch
----------
Second way of doing it would be to soft reset the current Branch (sumant) to the latest version of the master branch(WARNING soft!!! i needed here or the changes will be deleted). Then rebase the branch with master and make 2 new commits
i would prefer the new branch method this is how is done usually
we can have as many branches as we want and each one shows a pull request we tried to do
later on we can delete these if we want or not

**Comment by NektariosFifes** [comment]
good job

**Review by NektariosFifes — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -31,6 +31,11 @@ Usage
 ```
 npm run desktop
 ```
+[Dowload mqtt.zip](https://github.com/asyncapi/simulator/files/13696873/mqtt.zip)
```
Comment (NektariosFifes) [inline README.md:34]: Can you put and explanation of what this file is.

Like: Example asyncApi file used in the video

File: README.md [hunk 2]
```diff
@@ -31,6 +31,11 @@ Usage
...
+
```
Comment (NektariosFifes) [inline README.md:35]: Can you put a title here indicating this is a showcase video? Also place this above the file link.

**Review by NektariosFifes — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SumantxD) [reply README.md:35]: Done !! :tada:

File: README.md (hunk 1 above)
Comment (SumantxD) [reply README.md]: Done !! :tada:

**Review by NektariosFifes — APPROVED** [review]

**Pull request merged**